Model/User.php

```diff
@@ -120,7 +120,6 @@
      */
     public function __construct()
     {
-        $this->salt = base_convert(sha1(uniqid(mt_rand(), true)), 16, 36);
```

Comment:
Yes, but the member var is still present, so after instantiating the class, `$this->salt` should still be null, shouldn't it?
You don't have to, it was just an idea